cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
+  },
+  video: true,
+  viewportHeight: 1920,
```

Comment:
The 'viewportHeight' is set to 1920, which is unusually wide. Typically, this setting is for the height of the viewport. You might have intended to use 1080 for 'viewportHeight' and 1920 for 'viewportWidth' instead.